extension/README.md

```diff
@@ -3,32 +3,45 @@
 
 The Aspire VS Code extension provides a set of commands and tools to help you work with Aspire and Aspire AppHost projects directly from Visual Studio Code.
 
-## Features
+## Commands
 
-- **Run**: Run an Aspire app host in development mode.
-- **Add**: Add an integration to the Aspire project.
-- **New**: Create a new Aspire project.
-- **Publish**: Generates deployment artifacts for an Aspire app host project. (Preview)
-- **Config**: Manage configuration settings.
-- **Deploy**: Deploy an Aspire app host project to its supported deployment targets. (Preview)
+The extension adds the following commands to VS Code:
 
-## Commands
+| Command                               | Description                                                                                                                                              | Availability |
+|---------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| Aspire: Add an integration            | Add an integration, such as the Aspire Redis integration (`Aspire.Hosting.Redis`) to the Aspire project.                                                 | Available    |
+| Aspire: Configure launch.json         | Adds the default Aspire debugger launch configuration to your workspace's `launch.json`, which will detect and run the apphost project in the workspace. | Available    |
+| Aspire: Deploy apphost                | Deploy the contents of an Aspire apphost to its defined deployment targets.                                                                              | Preview      |
+| Aspire: Manage configuration settings | Manage configuration settings.                                                                                                                           | Available    |
+| Aspire: New Aspire project            | Create a new Aspire project.                                                                                                                             | Available    |
+| Aspire: Open Aspire terminal          | Open an Aspire terminal for working with Aspire projects.                                                                                                | Available    |
```

Comment:
What is an Aspire Terminal? one that already is running?

Reply:
It's a special vs code terminal! in which commands are run. I updated the string